packages/react-search-ui/README.md

```diff
@@ -282,44 +478,321 @@ import { Sorting } from "@elastic/react-search-ui";
+| `addFilter`         | `name` String - field name to filter on<br/>`value` String - field value to filter on                                                                                      |        | Add a filter in addition to current filters values                                          |
+| `setFilter`         | `name` String - field name to filter on<br/>`value` String - field value to filter on                                                                                      |        | Set a filter value, replacing current filter values                                         |
+| `removeFilter`      | `name` String - field name to filter on<br/>`value` String - field value to filter on                                                                                      |        | Remove a single filter value                                                                |
+| `clearFilters`      |                                                                                                                                                                            |        | Clear all filters                                                                           |
+| `setCurrent`        | Integer                                                                                                                                                                    |        |                                                                                             |
+| `setResultsPerPage` | Integer                                                                                                                                                                    |        |                                                                                             |
+| `setSearchTerm`     | String                                                                                                                                                                     |        |                                                                                             |
+| `setSort`           | `sortField` String - field to sort on<br/>`sortDirection` String - "asc" or "desc"                                                                                         |        |                                                                                             |
+| `trackClickThrough` | `documentId` String - The document ID associated with the result that was clicked<br/>`tag` - Array[String] Optional tags which can be used to categorize this click event |        | Report a click through event. A click through event is when a user clicks on a result link. |
```

Comment:
We seem to refer to a clickthrough as a single word in [the Site Search docs](https://swiftype.com/documentation/site-search/analytics#recording_clickthroughs) and other places.